applications/geoportal/index.js

```diff
@@ -40,6 +40,8 @@ jQuery(document).ready(function () {
                 return;
             }
 
+            appSetup.startupSequence.push({'bundlename': 'inspire'});            
```

Comment:
Remove this ^ lets update it to db instead.

Reply:
You can try running the bundle from dev-console with: `Oskari.app.playBundle({'bundlename': 'inspire'});`